spec/locale-sensitive-functions.html

```diff
@@ -32,7 +32,7 @@ <h1>String.prototype.localeCompare ( _that_ [ , _locales_ [ , _options_ ] ] )</h
       </emu-alg>
 
       <p>
-        The value of the `length` property of the *localeCompare* method is 1.
+        The value of the *"length"* property of the *localeCompare* method is 1.
```

Comment:
Here's another method that could be updated to backticks (though I guess that may go beyond the title of this PR).

Reply:
Making a new PR for these.